client/modules/App/components/Overlay.jsx

```diff
@@ -30,25 +59,46 @@ class Overlay extends React.Component {
       children
     } = this.props;
     return (
-      <div className="overlay">
-        <div className="overlay__content">
-          <section
-            tabIndex="0"
-            role="main"
-            aria-label={ariaLabel}
-            ref={(element) => { this.overlay = element; }}
-            className="overlay__body"
-          >
-            <header className="overlay__header">
-              <h2 className="overlay__title">{title}</h2>
-              <button className="overlay__close-button" onClick={this.close}>
-                <InlineSVG src={exitUrl} alt="close overlay" />
-              </button>
-            </header>
-            {children}
-          </section>
+      <a tabIndex={-42} onKeyDown={this.keyPressHandle} id="overlay__anchor">
```

Comment:
i think there's no difference between setting the tab index to -1 and to any other negative integer, right?

Reply:
also is this valid/semantic html? can you put a div inside of an a tag?

Reply:
Since i am giving the anchor a keypress property i guess it does have a weak semantic meaning. It's the only hack I know to put up with Handling events since divs can't have event handling in react.